R/forest_plot.R

```diff
@@ -121,8 +122,8 @@ forest_plot <- function(df, facet_by = NA, shape_by = NA, col_by = NA,
     ## use the palette if provided, otherwise use the default
     ## as defined in epireview
     ## if neither is provided, use the default palette
-    if (!is.na(col_palette)) {
-      p <- p + scale_color_manual(values = col_palette, na.value = "gray")
+    if (!is.null(col_palette)) {
+      p <- p + scale_color_manual(values = col_palette,  na.value = "gray")
```

Comment:
@shaziaruybal Same as my previous comment, can you check that this na.value param is fine?

Reply:
@tristan-myles and I have checked and we think the na.value param is fine